style.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css2?family=Poiret+One&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+    font-family: 'Poiret One', cursive;
+    font-weight: bold;
+}
+
+body{
+    display: flex;
```

Comment:
```suggestion
    display: grid
    place-items: center;
```

Reply:
done!!!

but is there any advantage of using grid instead of flex?